apis/core.oam.dev/common/types.go

```diff
@@ -327,9 +327,10 @@ type PolicyStatus struct {
 
 // WorkflowStatus record the status of workflow
 type WorkflowStatus struct {
-	AppRevision string       `json:"appRevision,omitempty"`
-	Mode        WorkflowMode `json:"mode"`
-	Message     string       `json:"message,omitempty"`
+	AppRevision  string       `json:"appRevision,omitempty"`
+	Mode         WorkflowMode `json:"mode"`
+	Message      string       `json:"message,omitempty"`
+	ResetCounter bool         `json:"resetCounter,omitempty"`
```

Comment:
how does user know the `Counter` mean? Does it mean loop backoff for users?

Reply:
> how does user know the `Counter` mean? Does it mean loop backoff for users?

Yes. This field is designed to be used by Vela CLI to trigger immediate retry after long waiting. It is not necessary for user to understand it.